src/gsi/gsi/gsiTypes.h

```diff
@@ -35,6 +35,7 @@
 #include <set>
 #include <stdexcept>
 #include <cstdint>
+#include <optional>
```

Comment:
doesn't that need to be guarded against older C++?

Reply:
Yes, will be adjusted.